### ISSUE TITLE
Consider adding caveats for service access keys

### ISSUE THREAD

**Issue by aegershman** [cloudfoundry/docs-dev-guide#336]
[As a CF operator I'm concerned about the message of service access keys:](https://github.com/cloudfoundry/docs-dev-guide/blob/master/services/index.html.md.erb#L34)

>For instance, you can use service keys to connect to a service instance from a local client, or from an app in another space, or even from outside of Cloud Foundry

- When connecting services cross-spaces, it's likely best to use service-instance sharing between spaces.
- With service instances which use bosh-dns for hostname resolution, any services which connects from off-platform won't be able to resolve the bosh-dns hostname string.
- Having longstanding credentials living in a service key reduces your security position because the rotation of those credentials are now a manual, static process.

I'm of the mindset that service keys should be used very sparingly and should be generally discouraged. We have developers who use service access keys for hooking up their data to external services or for integration tests; we're trying to move them off using keys for these services, but it's slow; our ability to take updates for services which take advantage of bosh-dns / bosh-hotswap will be limited by this.

Just wanted to share these problems so that someone might consider better articulating them in the form of a warning/caveat in the doc.

**Comment by jncd** [comment]
Hi @aegershman 
We noticed that you closed this issue earlier this month. 
Do you still want us to look into it?

**Comment by aegershman** [comment]
Nah, it's all good, it's really not a big deal. Thank you though 👍
